go.mod

```diff
@@ -3,79 +3,82 @@ module github.com/ComplianceAsCode/ocp4e2e
 go 1.23
 
 require (
-	github.com/cenkalti/backoff/v4 v4.1.1
```

Comment:
We'll have to try adding the `toolchain` back in once we start using a newer version of golang.